
[2020-11-12 19:47:56 +0000] [149] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['0.0.0.0:8080']
  backlog: 2048
  workers: 1
  worker_class: gthread
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /app
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 0
  group: 0
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: None
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: gunicorn_error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f7237a5e700>
  on_reload: <function OnReload.on_reload at 0x7f7237a5e820>
  when_ready: <function WhenReady.when_ready at 0x7f7237a5e940>
  pre_fork: <function Prefork.pre_fork at 0x7f7237a5ea60>
  post_fork: <function Postfork.post_fork at 0x7f7237a5eb80>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f7237a5eca0>
  worker_int: <function WorkerInt.worker_int at 0x7f7237a5edc0>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f7237a5eee0>
  pre_exec: <function PreExec.pre_exec at 0x7f7237a73040>
  pre_request: <function PreRequest.pre_request at 0x7f7237a73160>
  post_request: <function PostRequest.post_request at 0x7f7237a731f0>
  child_exit: <function ChildExit.child_exit at 0x7f7237a73310>
  worker_exit: <function WorkerExit.worker_exit at 0x7f7237a73430>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f7237a73550>
  on_exit: <function OnExit.on_exit at 0x7f7237a73670>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2020-11-12 19:47:56 +0000] [149] [INFO] Starting gunicorn 20.0.4
[2020-11-12 19:47:56 +0000] [149] [DEBUG] Arbiter booted
[2020-11-12 19:47:56 +0000] [149] [INFO] Listening at: http://0.0.0.0:8080 (149)
[2020-11-12 19:47:56 +0000] [149] [INFO] Using worker: gthread
[2020-11-12 19:47:56 +0000] [150] [INFO] Booting worker with pid: 150
[2020-11-12 19:47:56 +0000] [149] [DEBUG] 1 workers
[2020-11-12 19:48:12 +0000] [150] [DEBUG] GET /login
[2020-11-12 19:48:18 +0000] [150] [DEBUG] GET /login
[2020-11-12 19:48:30 +0000] [150] [DEBUG] GET /test
[2020-11-12 19:48:39 +0000] [150] [DEBUG] GET /test
[2020-11-12 19:48:54 +0000] [150] [DEBUG] GET /login
[2020-11-12 19:49:44 +0000] [150] [DEBUG] POST /login
[2020-11-12 19:51:17 +0000] [150] [DEBUG] POST /login
[2020-11-12 19:58:16 +0000] [149] [INFO] Handling signal: int
[2020-11-12 19:58:16 +0000] [150] [INFO] Worker exiting (pid: 150)
[2020-11-12 19:58:16 +0000] [149] [INFO] Shutting down: Master
[2020-11-12 19:58:19 +0000] [156] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['0.0.0.0:8080']
  backlog: 2048
  workers: 1
  worker_class: gthread
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /app
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 0
  group: 0
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: None
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: gunicorn_error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f3a0d394700>
  on_reload: <function OnReload.on_reload at 0x7f3a0d394820>
  when_ready: <function WhenReady.when_ready at 0x7f3a0d394940>
  pre_fork: <function Prefork.pre_fork at 0x7f3a0d394a60>
  post_fork: <function Postfork.post_fork at 0x7f3a0d394b80>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f3a0d394ca0>
  worker_int: <function WorkerInt.worker_int at 0x7f3a0d394dc0>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f3a0d394ee0>
  pre_exec: <function PreExec.pre_exec at 0x7f3a0d3a9040>
  pre_request: <function PreRequest.pre_request at 0x7f3a0d3a9160>
  post_request: <function PostRequest.post_request at 0x7f3a0d3a91f0>
  child_exit: <function ChildExit.child_exit at 0x7f3a0d3a9310>
  worker_exit: <function WorkerExit.worker_exit at 0x7f3a0d3a9430>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f3a0d3a9550>
  on_exit: <function OnExit.on_exit at 0x7f3a0d3a9670>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2020-11-12 19:58:19 +0000] [156] [INFO] Starting gunicorn 20.0.4
[2020-11-12 19:58:19 +0000] [156] [DEBUG] Arbiter booted
[2020-11-12 19:58:19 +0000] [156] [INFO] Listening at: http://0.0.0.0:8080 (156)
[2020-11-12 19:58:19 +0000] [156] [INFO] Using worker: gthread
[2020-11-12 19:58:19 +0000] [157] [INFO] Booting worker with pid: 157
[2020-11-12 19:58:19 +0000] [156] [DEBUG] 1 workers
[2020-11-12 19:58:33 +0000] [157] [DEBUG] POST /login
[2020-11-12 19:59:04 +0000] [156] [INFO] Handling signal: int
[2020-11-12 19:59:04 +0000] [157] [INFO] Worker exiting (pid: 157)
[2020-11-12 19:59:04 +0000] [156] [INFO] Shutting down: Master
[2020-11-12 19:59:13 +0000] [162] [DEBUG] Current configuration:
  config: gunicorn.conf.py
  bind: ['0.0.0.0:8080']
  backlog: 2048
  workers: 1
  worker_class: gthread
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /app
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 0
  group: 0
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: None
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: gunicorn_error.log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f0867004700>
  on_reload: <function OnReload.on_reload at 0x7f0867004820>
  when_ready: <function WhenReady.when_ready at 0x7f0867004940>
  pre_fork: <function Prefork.pre_fork at 0x7f0867004a60>
  post_fork: <function Postfork.post_fork at 0x7f0867004b80>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f0867004ca0>
  worker_int: <function WorkerInt.worker_int at 0x7f0867004dc0>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f0867004ee0>
  pre_exec: <function PreExec.pre_exec at 0x7f0867019040>
  pre_request: <function PreRequest.pre_request at 0x7f0867019160>
  post_request: <function PostRequest.post_request at 0x7f08670191f0>
  child_exit: <function ChildExit.child_exit at 0x7f0867019310>
  worker_exit: <function WorkerExit.worker_exit at 0x7f0867019430>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f0867019550>
  on_exit: <function OnExit.on_exit at 0x7f0867019670>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2020-11-12 19:59:13 +0000] [162] [INFO] Starting gunicorn 20.0.4
[2020-11-12 19:59:13 +0000] [162] [DEBUG] Arbiter booted
[2020-11-12 19:59:13 +0000] [162] [INFO] Listening at: http://0.0.0.0:8080 (162)
[2020-11-12 19:59:13 +0000] [162] [INFO] Using worker: gthread
[2020-11-12 19:59:13 +0000] [163] [INFO] Booting worker with pid: 163
[2020-11-12 19:59:13 +0000] [162] [DEBUG] 1 workers
[2020-11-12 19:59:49 +0000] [163] [DEBUG] GET /
[2020-11-12 19:59:49 +0000] [163] [DEBUG] GET /favicon.ico
[2020-11-12 20:00:04 +0000] [163] [DEBUG] GET /
[2020-11-12 20:01:28 +0000] [163] [DEBUG] POST /login
[2020-11-12 20:01:28,432] ERROR in app: Exception on /login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/app/auth.py", line 54, in login
    if not check_user_exist(**data):
TypeError: check_user_exist() got an unexpected keyword argument 'utzg6grkve'
[2020-11-12 20:01:56 +0000] [163] [DEBUG] POST /login
[2020-11-12 20:01:56,517] ERROR in app: Exception on /login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/app/auth.py", line 54, in login
    if not check_user_exist(**data):
TypeError: check_user_exist() got an unexpected keyword argument 'mu30y4eoo4'
[2020-11-12 20:02:23 +0000] [163] [DEBUG] POST /login
[2020-11-12 20:02:23,395] ERROR in app: Exception on /login [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/app/auth.py", line 54, in login
    if not check_user_exist(**data):
TypeError: check_user_exist() got an unexpected keyword argument 'u4pi2ou6ymq'
[2020-11-12 20:06:05 +0000] [163] [DEBUG] GET /
[2020-11-12 20:06:06 +0000] [163] [DEBUG] GET /favicon.ico
[2020-11-12 20:06:12 +0000] [163] [DEBUG] GET /
[2020-11-12 20:07:38 +0000] [163] [DEBUG] POST /login
[2020-11-12 20:09:14 +0000] [163] [DEBUG] worker: SIGWINCH ignored.
[2020-11-12 20:09:14 +0000] [162] [INFO] Handling signal: winch
[2020-11-12 20:09:14 +0000] [162] [DEBUG] SIGWINCH ignored. Not daemonized
[2020-11-12 20:09:15 +0000] [163] [DEBUG] worker: SIGWINCH ignored.
[2020-11-12 20:09:15 +0000] [162] [INFO] Handling signal: winch
[2020-11-12 20:09:15 +0000] [162] [DEBUG] SIGWINCH ignored. Not daemonized
[2020-11-12 20:09:27 +0000] [163] [DEBUG] worker: SIGWINCH ignored.
[2020-11-12 20:09:27 +0000] [162] [INFO] Handling signal: winch
[2020-11-12 20:09:27 +0000] [162] [DEBUG] SIGWINCH ignored. Not daemonized
[2020-11-12 20:09:31 +0000] [162] [INFO] Handling signal: int
[2020-11-12 20:09:31 +0000] [163] [INFO] Worker exiting (pid: 163)
[2020-11-12 20:09:31 +0000] [162] [INFO] Shutting down: Master
